testing/endtoend/evaluators/operations.go

```diff
@@ -88,6 +110,10 @@ var SubmitWithdrawal = e2etypes.Evaluator{
 	Name: "submit_withdrawal_epoch_%d",
 	Policy: func(currentEpoch primitives.Epoch) bool {
 		fEpoch := params.BeaconConfig().CapellaForkEpoch
+		// If Capella is disabled (starting at Electra+), run after exit submission
```

Comment:
Can't this be `if e2etypes.GenesisFork() >= version.Deneb`? Also this comment is wonky, shouldn't it say `Deneb+`?